CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+- Our sprint cycle is about a month. In this month we meet twice online - once in the middle of the month to check-in and discuss problems, once at the end/beginning of the month to demo our progress of the sprint. We ask you to join at least the demo call once per month.
+- We use slack as our main communication tool and do asynchronous [stand-ups](https://en.wikipedia.org/wiki/Stand-up_meeting) there once a week.
+- On Fridays, we offer a space for troubleshooting if you are stuck and need to dicuss something about your task.
+
+### How do I get my tasks? Where do I find information about them?
+We are using trello for Task Management. Our [product manager](https://www.youtube.com/watch?v=502ILHjX9EE) assigns the tickets to everybody taking part in a sprint. Each ticket should include a user story to understand why we need the ticket. If you are missing information, do not feel shy to write in slack.
+
+### How do I start with my task?
+We would like to do [test driven development](https://www.browserstack.com/guide/tdd-vs-bdd-vs-atdd) where possible. Thinking about the tests first forces you to think about a clear image how a screen or endpoint must look like.
+Besides that, check our [guidelines](#guidelines) and [development principles](#development-principles).
+
+### I'm stuck. How do I get help?
+Basically write in the #dev channel in slack or come to the troubleshooting meeting on Friday mentioned above.
+Please write in **public** channels and tag people instead of writing them privately. It is completely ok to be stuck and ask for help and this way others with the same problem find the solution easier.
+
+If you want somebody to look at your code when you are stuck, then [**create a draft Pull Request**](https://github.blog/2019-02-14-introducing-draft-pull-requests/). It gives the others a quick insight in what you have done and is great to directly comment on the code.
+
+### I'm finished with my task. What now?
+If you haven't used GitHub before, [this is a tutorial](http://makeapullrequest.com/) that will help. Don't be afraid to ask for help, if you need it. (Everyone is a beginner at some point!)
+
+- Commit all your progress on a branch which you started from master. [Here, guidelines how to write good commit messages](https://github.com/erlang/otp/wiki/writing-good-commit-messages).
+- Push your branch to github.
+- Create a [Pull Request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). The PR description should include an explanation of *why* you ended up with the solution you did and the intended behavior. Be clear, especially if you had to make a ‘call’ diverging from the description. 
+- Try to make it as easy as possible for the reviewer so that she can get back to you quickly. **Screenshots** and **step-by-step** guides are especially helpful with big PRs.
```

Comment:
"**she** can get back to you" ?

Reply:
It is a German approach in some newspapers to mix masculine and feminine generic pronouns to write gender neutral. Not sure if that's the correct way in English. @aerinsol

Reply:
I'de probably write "so that they can get back to you".